internal/schema/schema_test.go

```diff
@@ -834,6 +835,8 @@ Second line of the description.
 			}
 
 			union Union @uniondirective = Photo | Person
+
+			scalar Repeatable @repeatabledirective @repeatabledirective
```

Comment:
I'd rather use some other name for this scalar as I don't want to repeat the word `Repeatable`.

Reply:
😛 👍